.gitignore

```diff
@@ -48,3 +48,5 @@ gradle-app.setting
 .idea/
 *.iml
 out/
+/bin/
+
```

Comment:
The build failed due to a temporary network error - I've helped you restart it. You can check the build log to see the cause of failure; it isn't always your fault haha :)